lib/mayaUsd/ufe/UsdAttribute.cpp

```diff
@@ -532,14 +550,13 @@ Ufe::Value UsdAttribute::getMetadata(const std::string& key) const
                 return Ufe::Value(ss.str());
             }
         }
-        return Ufe::Value();
```

Comment:
Do not return immediately here. There might be information coming from the AttributeDef.